resources/email_to_class.txt

```diff
@@ -24,6 +24,8 @@ You should have the following software installed before the first class:
 2. Please install the latest version of RStudio (Desktop): https://posit.co/download/rstudio-desktop/
 You can find detailed instructions on how to do this on the <website page for Day 0>. Note that if you have any restrictions on your work computer, you might need to work with your IT department to get R and RStudio installed.
 
+To get the most out of this part of the program, if possible, we suggest downloading Zoom and RStudio on the same computer, so that you can copy and paste your code if needed or share your code form your screen.
```

Comment:
```suggestion
To get the most out of this part of the program, if possible, we suggest downloading Zoom and RStudio on the same computer, so that you can copy and paste your code if needed or share your code from your screen.
```